Automattic/wc-calypso-bridge
# Add phpcs and format files

This PR adds in phpcs linting and formats a number of the include files.  WPCS rulesets are automatically pulled in and set for phpcs.

<img width="618" alt="screen shot 2018-10-25 at 1 55 32 pm" src="https://user-images.githubusercontent.com/10561050/47523549-efb56200-d85d-11e8-9baf-1c42049f3f74.png">

### Testing

1.  First run `composer install` from your terminal
2.  Run `composer phpcs .` and note errors/warnings for files
3.  Verify no regressions or broken functionality from class renaming.

### Remaining tasks

Here are some other issues that might be best broken down into other PRs.

* Doc blocks are needed for the classes and all files under `includes/connect`
* `store-on-wpcom/` still needs formatting.
* Adding a pre-commit hook to test for lint errors.
